tests/ComposerLockTest.php

```diff
@@ -51,10 +52,10 @@ public function seesNonDevPlugins(): void
      */
     public function seesDevPlugins(): void
     {
-        $lock = new ComposerLock([$this->jsonFile((object)[
+        $lock = new ComposerLock([$this->jsonFile([
             'packages' => [],
             'packages-dev' => [
-                (object) $this->pluginEntry('vendor/package', 'Vendor\Package\PluginClass'),
+                $this->pluginEntry('vendor/package', 'Vendor\Package\PluginClass'),
```

Comment:
I'm not sure why `object` was used here, but both objects and keyed-arrays are encoded as objects in JSON, so i assume it's redundant.

Reply:
I vaguely recall something breaking (perhaps it was empty object vs empty array) so I just used `(object)[...]` as a cheap replacement for object literal PHP lacks.

Reply:
Okay, i reverted this changes just in case